images/macos/software-report/SoftwareReport.Generator.ps1

```diff
@@ -40,7 +40,7 @@ $languageAndRuntime.AddNodes($(Get-GccVersions))
 $languageAndRuntime.AddNodes($(Get-FortranVersions))
 $languageAndRuntime.AddToolVersion("Julia", $(Get-JuliaVersion))
 $languageAndRuntime.AddToolVersion("Kotlin", $(Get-KotlinVersion))
-if (-not $os.IsVentura) {
+if (-not $os.IsVentura -and -not $os.IsVenturaArm64) {

```

Comment:
@Alexey-Ayupov it should be `-or`?

Reply:
No, with -or we will get true on one of the OSes, we need to exclude both. I will rewrite condition in the next manner for better understandability ((-not $os.IsVentura) -and (-not $os.IsVenturaArm64))